Fix:
warning: array index 20 is past the end of the array (which contains 20 elements) [-Warray-bounds]
--- unzoo.c.orig	2002-05-06 10:34:00.000000000 -0500
+++ unzoo.c	2022-01-05 07:21:42.000000000 -0600
@@ -1461,7 +1466,7 @@
 int             DescReadArch ()
 {
     /* read the text at the beginning                                      */
-    BlckReadArch(Descript.text,20L);  Descript.text[20] = '\0';
+    BlckReadArch(Descript.text,20L);  Descript.text[19] = '\0';
 
     /* try to read the magic words                                         */
     if ( (Descript.magic = WordReadArch()) != (unsigned long)0xfdc4a7dcL )
